plugins/tech-radar/src/components/RadarPage.tsx

```diff
@@ -74,9 +74,16 @@ export function RadarPage(props: TechRadarPageProps) {
             onChange={e => setSearchText(e.target.value)}
           />
           <SupportButton>
-            This is used for visualizing the official guidelines of different
-            areas of software development such as languages, frameworks,
-            infrastructure and processes.
+            <p>
+              This is used for visualizing the official guidelines of different
+              areas of software development such as languages, frameworks,
+              infrastructure and processes. You can find an explanation for the
+              radar at{' '}
+              <Link href="https://opensource.zalando.com/tech-radar/">
```

Comment:
This should probably have `target="_blank" rel="noopener noreferrer"` on it to avoid taking the user out of their context in Backstage.

Reply:
Rather, please switch over to using Link from `@backstage/core-components` instead. It has this behavior our of the box.

Reply:
Switched to `core-components`.